app/public/templates/scene_nav_mesh.html

```diff
@@ -13,7 +13,7 @@
       <a-scene
         cursor="rayOrigin:mouse"
         raycaster="objects: .link"
-        inspector-plugin-recast
+        inspector-plugin-recast="serviceURL: http://localhost:3000;"
```

Comment:
Should we use a variable and define it somewhere that will be easy to change later?

Reply:
This one is just a file that we use to generate the navmesh that won't be exposed or rendered in our actual application. So it is okay (rather I would prefer) to leave it as is, rather than putting it in a config variable.